packages/fhir-heatlhcareservice/src/components/HealthCareList/index.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { ChangeEvent, useMemo, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Row, Col, Button, Input } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+import HealthCaresDetail from '../HealthCareDetail';
+import { SearchOutlined } from '@ant-design/icons';
+import { sendErrorNotification } from '@opensrp/notifications';
+import { HealthcareService, HealthcareServiceDetail } from '../../types';
+import { HEALTHCARES_GET, URL_ADD_HEALTHCARE } from '../../constants';
+import Table from './Table';
+import './index.css';
+import { Spin } from 'antd';
+import { Link } from 'react-router-dom';
+import lang from '../../lang';
+import { useQuery } from 'react-query';
+import FHIR from 'fhirclient';
+import { loadHealthcareOrganization } from '../../utils';
+import { FHIRResponse } from 'react-utils';
+
+interface Props {
+  fhirBaseURL: string;
+}
+
+/** Function which shows the list of all healthcares and there details
+ *
+ * @param {Object} props - HEALTHCAREsList component props
+ * @returns {Function} returns healthcare display
+ */
+export const HealthCareList: React.FC<Props> = (props: Props) => {
+  const { fhirBaseURL } = props;
+
+  const serve = FHIR.client(fhirBaseURL);
+
+  const [detail, setDetail] = useState<HealthcareServiceDetail | 'loading' | null>(null);
+  const [filterData, setfilterData] = useState<{ search?: string; data?: HealthcareService[] }>({});
+
+  const healthcare = useQuery(HEALTHCARES_GET, () => serve.request(HEALTHCARES_GET), {
```

Comment:
Refactor to use pagination params since the HAPI server returns a maximum of 20 resource entries